homeassistant/components/weatherflow/sensor.py

```diff
@@ -364,6 +365,11 @@ def __init__(
                 description.imperial_suggested_unit
             )
 
+    @property
+    def available(self) -> bool:
+        """Return True if entity is available."""
+        return self.native_value is not None
```

Comment:
The downside to implementing it this way is that every state write will calculate the native value twice and run the conversion function twice when it is available (probably the common case)

To avoid this we usually implement calculating both values (native value and available) where the data comes in